guides/v2.3/ui_comp_guide/components/ui-masonry.md

```diff
@@ -0,0 +1,102 @@
+---
+group: ui-components-guide
+title: Masonry component
+---
+
+The Masonry component is a collection of columns. It renders the masonry grid layout in the Ui component
+
+## Dependencies
+
+The Masonry component has dependencies on the following components:
+
+*  Listing: `app/code/Magento/Ui/view/base/web/js/grid/listing.js`
+*  Request Animation Frame: `app/code/Magento/Ui/view/base/web/js/lib/view/utils/raf.js`
+
+## Configuration options
+
+<table>
```

Comment:
I would suggest using Markdown for tables instead of HTML. Please, refer to the [following](https://devdocs.magento.com/guides/v2.3/contributor-guide/templates/basic_template.html#tables) topic for more details